code/controllers/subsystem/ticker.dm

```diff
@@ -452,9 +452,12 @@ var/datum/subsystem/ticker/ticker
 
 /datum/subsystem/ticker/proc/send_random_tip()
 	var/list/randomtips = file2list("config/tips.txt")
-	if(randomtips.len)
+	var/list/memetips = file2list("config/sillytips.txt")
+	if(randomtips.len && prob(95))
 		world << "<font color='purple'><b>Tip of the round: </b>[html_encode(pick(randomtips))]</font>"
-
+	else if(memetips.len)
+		world << "<font color='purple'><b>Tip of the round: </b>[html_encode(pick(memetips))]</font>"
```

Comment:
remove the string duplicattion
i.e pick value and store in temp var then do the string output once at the end of the proc

:+1: